content/updates/2021-11/2-TA-addrows.md

```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_img: ./img/TA_addrows.png
+pim_announcement_alt_img: "Table Attribute"
+pim_announcement_audience:
+- EE
+- GE
+---
+
+# Add New Rows from the Product Edit Form
+::: meta-data type="New" features="Governance" available="Late-October" in="EE" link-to-doc="../articles/manage-multidimensional-data-in-a-table.html"
+
+Add new rows directly from the product edit form as soon as you realize they are missing. You no longer have to go back to the Settings page. Staying in the PEF will help you stay focused on enrichment and without this context-switching, your products will be ready to be shared in no time!
```

Comment:
If product edit form is capitalized in the title, it should be capitalized here too